pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -810,6 +842,12 @@ def num_ipus(self) -> int:
             return self.devices
         return 0
 
+    @property
+    def num_hpus(self) -> int:
+        if isinstance(self.accelerator, HPUAccelerator):
+            return self.devices
+        return 0
```

Comment:
the properties below are marked for deprecation in favor of a generic `num_devices` property. le's not add a hpu-specific one here.